engine/src/main/java/org/destinationsol/game/sound/OggSoundManager.java

```diff
@@ -34,27 +34,73 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * This class represents the sound manager used in  DestinationSol. It is responsible for handling and playing all
```

Comment:
This is great info with some extra nice usage of javadoc tags, but overall it is a really dense wall of text, and that comes from somebody who loves walls of text! :-)

Split into some paragraphs if needed and consider a less conversational tone in favor of a more reference-based one. See https://github.com/Terasology/ModuleTestingEnvironment/blob/master/src/main/java/org/terasology/moduletestingenvironment/ModuleTestingEnvironment.java#L68 for a really good way to do a big description with a lot of nice formatting

An example reference-style rewrite might sound something like this:

"
Responsible for handling all game sounds. 

Loads sounds required from {@link Assets} in {@code .ogg} format in one of the asset-scanned directories. 

Optionally place a same-name text file as the ogg with a {@code .soundinfo} extension, containing:

<pre>
volume=x
loopTime=x
</pre>

{@code volume} specifies the default volume multiplier for the sound, as a {@code float} indicating a percentage
{@code loopTime} unset or zero will allow playing the sound again, even concurrently with itself. Greater than zero will only accept a replay after {@code float} x seconds from the same originator. 
"

(Note: I assume volume to be a percentage and loopTime being seconds, this may be wrong)

I believe that roughly conveys the same information but in fewer words with better formatting :-)

Reply:
Milliseconds, or even better, real-time undependent frame-based time units, not seconds :) But I get the point.